ffi/freetype.lua

```diff
@@ -184,28 +203,41 @@ function FTFace_mt.__index:getInfo()
     return finfo
 end
 
-local FTFaceType = ffi.metatype("struct FT_FaceRec_", FTFace_mt) -- luacheck: ignore 211
+function FTSize_mt.__index:done()
+    done_size(self)
+end
+
+local FTSizeType = ffi.metatype("struct FT_SizeRec_", FTSize_mt) -- luacheck: ignore 211
 
-function FT.newFace(filename, pxsize, faceindex)
+local faces_cache = setmetatable({}, {__mode="v"})
+
+function FT.newSize(filename, pxsize, faceindex)
```

Comment:
What do you think of naming this one `newFaceSize` ? Would read more explicitely in frontend that it's a face size object, rather than a "size" ?

```lua
ok, ftsize = pcall(Freetype.newFaceSize, _v, size, faceindex)
ok, ftsize = pcall(Freetype.newSizedFace, _v, size, faceindex) -- too strange?
```

Reply:
Done.